backend/python/app/conftest.py

```diff
@@ -24,24 +24,33 @@ def app():
     return my_app
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="function")
```

Comment:
default scope should be `function`, but declaring it here explicitly to avoid confusion